### ISSUE TITLE
Can't Upload Blueprint Book, Error Non Descriptive

### ISSUE THREAD

**Issue by RedRumNCoke** [barthuijgen/factorio-sites#382]
I am getting an ugly preview, and then an error uploading my blueprint book. I suspect it's because the book is massive, but some more helpful feedback would be great. 
![image](https://github.com/user-attachments/assets/fab2ef1d-27e8-4318-a4ca-9c8bca727ec0)

**Comment by RedRumNCoke** [comment]
> Any chance you could share the string so I can test it? 
Yes! I managed to get it uploaded on [Factoriobin](https://factoriobin.com/post/3vpk1k)

**Comment by barthuijgen** [comment]
Perfect thank you, I'll take a look. Can't guarantee any time but I'll leave this issue open until then.

**Comment by RedRumNCoke** [comment]
No rush, thanks for the help! I'd love to put this somewhere that has a better user experience and some actual previews and such :)
